ssg/build_yaml.py

```diff
@@ -539,6 +541,25 @@ def from_yaml(yaml_file, env_yaml=None):
         rule.validate_references(yaml_file)
         return rule
 
+    def to_contents_dict(self):
+        """
+        Returns a dictionary that is the same schema as the dict obtained when loading rule YAML.
+        """
+        interesting_keys = [
```

Comment:
@matejak can this be automated rather than being explicit?

Reply:
I have moved to semi-automated, I am somehow not in favor of automated, as they say that [explicit is better than implicit](https://www.python.org/dev/peps/pep-0020/).